server/index.js

```diff
@@ -0,0 +1,26 @@
+
+const express = require("express");
+const ethers = require('ethers');
+const app = express();
+const cors = require("cors");
+const utils = ethers.utils;
+
+const provider = new ethers.providers.JsonRpcProvider('http://40.114.122.81:8545')
+var faucetWallet = new ethers.Wallet('0x938c57c1b1e3c43407e86e38dddd4f4a92e90fb0c1d55a21c8c66e907cc84a2a');
```

Comment:
Private key on GitHub is not a good idea :innocent:

Reply:
fixed